rdt-open-reader/src/covidIndonesia/assets/json.form/rdt-capture-form.json

```diff
@@ -329,6 +329,7 @@
         "openmrs_entity_id": "",
         "text": "{{rdt-capture-form-original.step6.manual_expiration_date.text}}",
         "hint": "{{rdt-capture-form-original.step6.manual_expiration_date.hint}}",
+        "min_date": "today+1d",
```

Comment:
https://github.com/onaio/rdt-standard/blob/6e5628e18688f4b7481b7ce0d0fdabe47bbefd4d/rdt-open-reader/src/covidIndonesia/assets/json.form/sample-collection-form-original.json#L331 should be used instead.

Reply:
any example

Reply:
you can click on the link above